chef/data_bags/crowbar/bc-template-glance.json

```diff
@@ -54,6 +54,7 @@
       },
       "rbd": {
         "store_ceph_conf": "/etc/ceph/ceph.conf",
+        "store_admin_keyring": "/etc/ceph/ceph.client.admin.keyring",
```

Comment:
This default value for internal ceph deployment, I would prefer to leave it as it is.